src/Graphs.js

```diff
@@ -0,0 +1,145 @@
+//chart js imports
+import 'chart.js/auto';
+import { Chart } from 'react-chartjs-2';
+
+
+//! make a select option for years and create an onchange function that takes the value and uses chart.js update to show you the database by year
+
+const Graphs = ( {graphData, firebaseWaste} ) => {
+
+  //prepares empty arrays for monthly trend chart. 
+  const categories = {
+    landfill: new Array(12).fill(0),
+    recycling: new Array(12).fill(0),
+    organics: new Array(12).fill(0),
+  }
+
+  //groups firebase data into correct format for chart.js barChart
+  const barGrouper = (data) => {
+    for (let wasteEntry in data) {
+      const entry = data[wasteEntry];
+      const dest = categories[entry.wasteType.toLowerCase()]
+      const monthEntry = new Date(entry.date).getMonth()
+      const weight = entry.wasteWeight;
+      dest[monthEntry] += weight
+    }
+  }
+
+  barGrouper(firebaseWaste)
+
+  return(
+    <section className="graphs">
+      <div className="wrapper">
+      <h2>Graphs</h2>
+      <div className="chartHolder">
+        <div className="barWrapper">
+          {/* chartJs bar component + config options */}
+          <Chart 
```

Comment:
If chart.js allows it may be worthwhile considering putting data or options in an object outside of the <Chart> element and then referencing that object. Would potentially make the element easier to parse.